src/frontend/types/XAPI.ts

```diff
@@ -0,0 +1,57 @@
+export type contextExtensionsKey =
+  | 'length'
```

Comment:
If you make this an `enum` you can reuse the names as keys on the actual `InitializedContextExtensions` interface definition below through the computed key notation

Reply:
I removed them, it was useless to create this type.